compiler-rt/lib/asan/asan_shadow_setup.cpp

```diff
@@ -109,6 +109,15 @@ void InitializeShadowMemory() {
     ProtectGap(kShadowGap2Beg, kShadowGap2End - kShadowGap2Beg + 1);
     ProtectGap(kShadowGap3Beg, kShadowGap3End - kShadowGap3Beg + 1);
   } else {
+#  if SANITIZER_LINUX
```

Comment:
if the call is within this guard, why do we need a dummy for macos and windows?

Reply:
Done